.dockstore.yml

```diff
@@ -57,6 +57,7 @@ workflows:
        branches:
          - master
          - ah_var_store
+         - mmt_lock_load
```

Comment:
do you want this committed?

Reply:
not once i merge, but i'll remove it right before merging, so that it stays in dockstore in case i have to keep testing